rfd/xxxx-machine-id-anonymous-usage-metrics.md

```diff
@@ -0,0 +1,79 @@
+---
+authors: Noah Stride (noah.stride@goteleport.com)
+state: draft
+---
+
+# RFD XXXX - Machine ID Anonymous Agent Usage Metrics
+
+## Required Approvers
+
+* Engineering: @zmb3
+* Product: @klizhentas && @xinding33
+
+## What
+
+Collect anonymous usage information about invocations of the Machine ID agent
+from the agent itself.
+
+Out of scope is the collection of usage information from the Auth server.
+
+## Why
+
+It is currently difficult to determine the uptake of Machine ID by customers,
+and for what use-cases they use Machine ID for. This makes product decisions
+about determining areas of focus more difficult.
+
+Adding basic anonymous metrics to the `tbot` agent will provide helpful
+information on adoption without compromising privacy.
+
+## Details
+
+### Collecting events
+
+
+### Submitting events
+
+### Consent
```

Comment:
I think my only query on this is if we are certain we want to pursue opt-in rather than opt-out here.